core/src/main/java/greencity/controller/ManagementNotificationController.java

```diff
@@ -0,0 +1,76 @@
+package greencity.controller;
+
+import greencity.annotations.ApiPageable;
+import greencity.constants.HttpStatuses;
+import greencity.dto.NotificationTemplateDto;
+import greencity.dto.PageableDto;
+import greencity.service.notification.ManagementNotificationService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import springfox.documentation.annotations.ApiIgnore;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/admin/notification")
+@RequiredArgsConstructor
+public class ManagementNotificationController {
+    private final ManagementNotificationService notificationService;
+
+    /**
+     * {@inheritDoc}
+     */
+    @ApiOperation(value = "Get all notification templates")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @GetMapping
+    @ApiPageable
+    public ResponseEntity<PageableDto<NotificationTemplateDto>> getAll(@ApiIgnore Pageable pageable) {
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(notificationService.findAll(pageable));
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @ApiOperation("Update notification template")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @PostMapping(value = "/update")
```

Comment:
PUT method is more suitable for update. Also, according to restful spec you should keep @ PostMapping empty